wdio.conf.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const defaultWdioConfig = require('terra-toolkit/config/wdio/wdio.conf');
+
+const wdioConfig = defaultWdioConfig.config;
+
+if (process.env.npm_package_name !== 'terra-graphs') {
```

Comment:
The mocked package you added is named terra-graphs

Reply:
I don't think so. My plans for this mono repo are to only have two packages here:

* terra-graphs: the adapter for carbon into the terra ecosystem as well as the location for dev site documentation and wdio testing using dev site.
* carbon: the un-opinionated graphing library without any react dependencies.

By using a mono repo setup we can integrate with the terra documentation and testing ecosystem while honoring carbons history of being an un opinionated library. Tests will still be included on each PR, just in the terra-graphs package instead of being in the carbon package and there won't be any direct dependency on react or exported react components from the carbon package.

Long term we could consider breaking up terra-graphs into the different graph types, but for now I think this plan will give us good mileage.

And I think terra-graphs is the best name for the mono-repo and the package.